src/main/resources/static/schema.graphql

```diff
@@ -54,4 +61,40 @@ enum RegionType {
     LOCATION_6
     LOCATION_7
     LOCATION_8
+}
+# order
+type Order {
+    id: ID,
+    orderNumber: String,
+    requestedBy: String,
+    status: String,
+    reviewedByAPV: Boolean,
+    reviewedByPRCH: Boolean,
+    respondedBy: String,
+    orderItems: [OrderItem],
+    region: String,
+    location: String,
+    orderImage: String,
+    receiverId: String,
+    eventId: String,
+    eventNumber: String,
+    submissionDate: DateTime,
+    approverReplyDate: DateTime,
+    comments: String
+}
+type OrderItem {
+    id: ID,
+    name: String,
+    description: String,
+    image: String,
+    order: Order,
```

Comment:
This field should be ignored:
1. REST API hide this in response.
2. It could be endless to return the "order" and then see items in it which will include "order" again.